boards/arm/lpcxpresso54114_m0/lpcxpresso54114_m0.yaml

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2017, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+identifier: lpcxpresso54114
+name: NXP LPCXpresso54114
+type: mcu
+arch: arm
+ram: 64
```

Comment:
Should this be 32 since its pointing at sram2?

Reply:
Ok, fixed.